concepts/50 React Components/02 Create a DevExtreme Application/10 Layouts.md

```diff
@@ -0,0 +1,19 @@
+The application includes two layouts that differ in the toolbar location:
```

Comment:
I don't understand what you want to say. Are you saying there are two locations in the toolbar or two different layouts?

Reply:
There are two layouts, and the only difference between them is where the toolbar is located.

Reply:
```suggestion
The application includes two layouts. The only difference between them is where they are located on the toolbar.
```